README.md

```diff
@@ -51,6 +51,12 @@ http://geoservices.tamu.edu/Services/Geocode/WebService/GeocoderWebServiceHttpNo
 https://search.mapzen.com/v1/search?api_key=MY_API_KEY&text={{address}}
 ```
 
+Since this is just a Mustache template, we can also expand it to concatenate multiple fields:
```

Comment:
I try to avoid "just" in docs - how about:

If your addresses are broken up into multiple columns (e.g. a `street_address` column, a `city` column, and a `state` column), you can use them all together in a URL template:

`https://maps.googleapis.com/maps/api/geocode/json?address={{street_address}},{{city}},{{state}}&key=MY_API_KEY`